recipe/meta.yaml

```
@@ -28,7 +27,9 @@ requirements:
     - libcurl
     - netaddr
     - psutil
+    - pyspnego
     - python
```

let's make the python constraint explicit

```suggestion
    - python >=3.8.0
```